faiss-search/flow-query.yml

```diff
@@ -3,18 +3,15 @@ with:
   read_only: true
 pods:
   crafter:
-    yaml_path: yaml/craft.yml
-    replicas: $REPLICAS
+    uses: _forward
   encoder:
-    yaml_path: yaml/encode.yml
-    replicas: $REPLICAS
-  faiss_indexer:
-    image: jinaai/hub.executors.indexers.vector.faiss:latest
-    replicas: $REPLICAS
-    timeout_ready: 10000
-    yaml_path: yaml/query-chunk.yml
+    uses: yaml/encode.yml
+    parallel: $PARALLEL
+  indexer:
+    uses: jinaai/hub.executors.indexers.vector.faiss:latest
+    parallel: $PARALLEL
+    timeout_ready: 600000
+    uses_internal: yaml/indexer.yml
```

Comment:
this is the problem @YueLiu-jina , this must be  `yaml/query-indexer.yml`

Reply:
Yes I noticed, my bad when solving conflict